webpack/scenes/ContentViews/Details/Filters/CVErrataDateFilterContent.js

```diff
@@ -171,6 +171,15 @@ const CVErrataDateFilterContent = ({
                       {__('Bugfix')}
                     </p>
                   </SelectOption>
+                  <SelectOption
+                    isDisabled={!hasPermission(permissions, 'edit_content_views')}
```

Comment:
If a user can't do something due to permissions, it should be hidden, not disabled.

Reply:
I was following the standard for the other three options in this menu which also disable on lack of permissions (or singleselection which wasn't applicable). I think it makes sense to either keep it disabled or change all the options to hidden. Which would you prefer?

Reply:
Ah, I didn't see the others. I think the general rule should still apply, and we should change all of them to hidden. But I will defer to @sjha4

Reply:
That sounds good to me!